software/controller/lib/hal/spi_chain.h

```diff
@@ -82,7 +82,7 @@ class RequestQueue {
   void AdvanceToNextRequest();
 };
 
-template <size_t MaxSlaves, size_t MaxRequestsPerSlave>
+template <size_t MaxSlaves, size_t QueuesLength>
```

Comment:
Maybe just `QueueLength`?

Reply:
I think it may be a peculiarity of English, where yes there are multiple queues, but then when something is "of queues" then the grammatically null-terminated form is used, so queue. I really miss having access to plural genitive declension in such situations :)